### DISCUSSION TITLE
please need help on template

### DISCUSSION THREAD

**Discussion by kurtdefeyter** [wlcrs/huawei_solar#481] [General]
Hi all,
I am trying to use this example code from the wiki
# Real house load
      #
      - name: house_consumption_power
        unique_id: house_consumption_power
        #friendly_name: "Power meter House power"
        state_class: measurement
        device_class: power
        unit_of_measurement: "W"
        icon: mdi:solar-power
        state: >-
          {% set inv_active_power = states('sensor.inverter_active_power')|float(0) %}
          {% set pm_active_power = states('sensor.power_meter_active_power')|float(0) %}
          {{ (inv_active_power - pm_active_power)|float(0)|round(0) }}
        availability: >-
          {{ (states('sensor.inverter_active_power')|is_number)
              and (states('sensor.power_meter_active_power')|is_number) }}

This is working fine however i get the value and always the state true 
Question is how i can get rif of true 

![template](https://github.com/wlcrs/huawei_solar/assets/125728797/ee8e8eb3-871e-479a-8d8c-25ac69f05be3)

**Comment by Roving-Ronin** [comment]
@kurtdefeyter 
Ahhh... delete the section calling for the availability :-)
Just use:

```
  #
  - name: house_consumption_power
    unique_id: house_consumption_power
    #friendly_name: "Power meter House power"
    state_class: measurement
    device_class: power
    unit_of_measurement: "W"
    icon: mdi:solar-power
    state: >-
      {% set inv_active_power = states('sensor.inverter_active_power')|float(0) %}
      {% set pm_active_power = states('sensor.power_meter_active_power')|float(0) %}
      {{ (inv_active_power - pm_active_power)|float(0)|round(0) }}
```

**Comment by kurtdefeyter** [comment]
wow , many thanks ...working now
